res/xml/user_details_settings.xml

```diff
@@ -25,22 +25,18 @@
             android:key="enable_calling"
             android:icon="@drawable/ic_phone"
             android:title="@string/user_enable_calling_sms" />
+    <com.android.settingslib.RestrictedPreference
```

Comment:
It's better to place this setting right before `app_copying` setting, they are both related to app installation.